### DISCUSSION TITLE
Is it possible to adjust grid cell sizes?

### DISCUSSION THREAD

**Discussion by aiem2018** [uber/h3#596] [Q&A] [answered]
Hi all, the library looks very useful, but for my purpose it would be more suitable to use smaller cell sizes (approximately 0.7x). So, my question:  is it possible to make some simple adjustment of  the root grid layer (size and number of cells) to obtain the corresponding cell size changes in all the derivative layers?

**Comment by dfellis** [comment, marked as answer]
@aiem2018 not really. The base cells were precomputed and hardwired, and there's assumptions throughout the codebase about them and how they're subdivided. This produces advantages in computing them and in sharing data between disparate systems: the H3 index value from multiple projects can be consumed together by another project and that third project can be sure that the same index from both actually refers to the exact same part of the planet.

What you probably want is [DGGRID](https://github.com/sahrk/DGGRID) which this project shares a history with. DGGRID is more flexible and configurable, with multiple subdivision rules possible, but due to that flexibility data indexed within it is less "shareable" in the ways described above with H3.

**Reply by aiem2018** [reply]
Thanks a lot!
